### PR TITLE
Replace some C++98isms with C++11isms (#816)

### PR DESCRIPTION
**Pull request by jpnurmi**

### CODE REVIEW

**Comment by DarthGandalf** [comment]
Thinking about override more... May you please remove "virtual" from
methods with "override" while you're doing this?
It makes it easy to "grep virtual" to find places where virtual hierarchy
starts.

2015-02-25 0:28 GMT-08:00 J-P Nurmi <notifications@github.com>:

**Comment by jpnurmi** [comment]
Good idea - done

**Comment by md-5** [comment]
Interesting, I've always just used lowercase null rather than nullptr.

**Comment by DarthGandalf** [comment]
@md-5 what is lowercase null? Uppercase `NULL` is older way to specify null pointer, `nullptr` is the new way.

File: modules/autocycle.cpp [hunk 1]
```diff
@@ -31,7 +31,7 @@ class CAutoCycleMod : public CModule {
 
 	virtual ~CAutoCycleMod() {}
```
Comment (DarthGandalf) [inline modules/autocycle.cpp]: This could be override too :)

File: modules/keepnick.cpp [hunk 2]
```diff
@@ -138,7 +138,7 @@ class CKeepNickMod : public CModule {
 		m_pTimer = NULL;
```
Comment (DarthGandalf) [inline modules/keepnick.cpp]: nullptr?

**Comment by DarthGandalf** [comment]
Did you use clang-modernize for this?

**Comment by jpnurmi** [comment]
Yes, clang-modernize was used. Interesting how it missed some NULL's.

**Pull request merged**